### PR TITLE
EMP-459 CRM 4 Defects - General Information Page

### PR DESCRIPTION
**Pull request by BNmoj**
changes -

- added boolean filter to utils.ts
- amended nunjucksSetup.ts
- amended formField.njk

### CODE REVIEW

File: server/views/components/formField.njk [hunk 1]
```diff
@@ -51,6 +51,8 @@
         <a href="{{ field.value.href }}" class="govuk-link govuk-link--no-visited-state">{{ field.value.text }}</a>
     {% elif field.type === 'every' %}
         {{ field.value | transformValue('every') }}
+    {% elif field.value === true or field.value === false %}
```
Comment (muyinatech) [inline server/views/components/formField.njk:54]: Need to also check for === 'true'  or === 'false'
If field.value = 'true' or 'false' string. Then it is not formatted Yes/No

File: server/utils/utils.ts [hunk 2]
```diff
@@ -42,3 +42,13 @@ export const currentIsoDate = (): string => {
 export const isNotEmpty = (value: string): boolean => {
   return value !== undefined && value !== null && value !== ''
 }
+
+export const formatBooleanToYesNo = (value: boolean | string | null | undefined): string => {
```
Comment (muyinatech) [inline server/utils/utils.ts:46]: Unit tests are missing

Comment (BNmoj) [reply server/utils/utils.ts]: tests added

File: server/utils/utils.ts [hunk 3]
```diff
@@ -42,3 +42,13 @@ export const currentIsoDate = (): string => {
...
+  if (value === true || value === 'true') {
+    return 'Yes'
+  }
+  if (value === false || value === 'false') {
+    return 'No'
+  }
+  return value ? value.toString() : ''
```
[the remark this answers is not in the document]
Comment (BNmoj) [reply server/utils/utils.ts]: amended

**Review by muyinatech — APPROVED** [review]

**Pull request merged**